source/rules/literals.rs

```diff
@@ -42,25 +42,26 @@ use nom::{
     hex_digit,
     oct_digit
 };
-use std::str;
 use std::str::FromStr;
+use std::str;
+use super::super::ast::Literal;
 
 named!(
-    pub null< Option<()> >,
+    pub null<Literal>,
     map_res!(
         tag!("null"),
```

Comment:
A new item has been added to list in https://github.com/tagua-vm/parser/issues/5.